python/src/aac/context/definition_parser.py

```diff
@@ -813,11 +813,16 @@ def load_definitions(
             for definition in self.parsed_definitions + self.context.get_definitions()
             if definition.get_root_key() == "primitive"
         }
-        self.fully_qualified_name_to_definition = {
-            definition.get_fully_qualified_name(): definition
-            for definition in self.parsed_definitions + self.context.get_definitions()
-            if "package" in definition.structure[definition.get_root_key()]
-        }
+        self.fully_qualified_name_to_definition = {}
+        for definition in self.parsed_definitions + self.context.get_definitions():
```

Comment:
For consistency, I would also adjust the logic structuring for the primitive type assignment above so that the flow and leverage is better understood/more readable